plugins/modules/sonic_logging.py

```diff
@@ -198,7 +199,7 @@
           remote_port: 622
           protocol: TCP
           source_interface: Ethernet24
-          message_type: audit
+          message_type: auditd-system
```

Comment:
Please show the corresponding changed result containing auditd-system in the "After state" display for this example.